EventFilter/L1TRawToDigi/plugins/implementations_stage2/GTSetup.cc

```diff
@@ -97,6 +102,7 @@ namespace l1t {
       muon_unp->setMuonCopy(amc - 1);
       egamma_unp->setEGammaCopy(amc - 1);
       etsum_unp->setEtSumCopy(amc - 1);
+      zdc_unp->setZDCSumCopy(amc - 1);
```

Comment:
```suggestion
      zdc_unp->setEtSumZDCCopy(amc - 1);
```
I would also change the name of this method.